lighthouse-core/audits/non-composited-animations.js

```diff
@@ -0,0 +1,145 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+const Audit = require('./audit.js');
+const i18n = require('../lib/i18n/i18n.js');
+
+const UIStrings = {
+  /** Title of a diagnostic LH audit that provides details on animations that are not composited. */
+  title: 'Avoid non-composited animations',
+  /** Description of a diagnostic LH audit that shows the user animations that are not composited. */
+  description: 'Animations which are not composited can be janky and contribute to CLS. ' +
+    '[Learn more](https://developers.google.com/web/fundamentals/performance/rendering/stick-to-compositor-only-properties-and-manage-layer-count)',
+  /** [ICU Syntax] Label identifying the number of animations that are not composited. */
+  displayValue: `{itemCount, plural,
+  =1 {# animated element found}
+  other {# animated elements found}
+  }`,
+};
+
+const str_ = i18n.createMessageInstanceIdFn(__filename, UIStrings);
+
+/**
+ * Each failure reason is represented by a bit flag. The bit shift operator '<<' is used to define which bit corresponds to each failure reason.
+ * https://source.chromium.org/chromium/chromium/src/+/master:third_party/blink/renderer/core/animation/compositor_animations.h;l=58;drc=f959aba6099c5d767f885b7f4632955b463dc41d?originalUrl=https:%2F%2Fcs.chromium.org%2F
+ * @type {{flag: number, text: string}[]}
+ */
+const ACTIONABLE_FAILURE_REASONS = [
```

Comment:
over in https://docs.google.com/document/d/1XKcJP2CKmNKfOcDsVvliAQ-e1H9C1nf2H-pzTdyafAA/edit?ts=5f233bae# adam and i started exploring if there are other failure reasons that are actionable.  
unsupported css prop is definitely the first to focus on. but looks like there are 1 or 3 others that may be useful.